Index: ../app/src/main/res/layout/activity_main.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ../app/src/main/res/layout/activity_main.xml	(date 1472006232000)
+++ ../app/src/main/res/layout/activity_main.xml	(revision )
@@ -25,4 +25,5 @@
         android:onClick="logout"
         android:layout_marginTop="50dp"
         android:text="注销" />
+
 </RelativeLayout>
Index: ../app/src/main/AndroidManifest.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ../app/src/main/AndroidManifest.xml	(date 1472006232000)
+++ ../app/src/main/AndroidManifest.xml	(revision )
@@ -15,7 +15,11 @@
 
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
-        </activity>
+        </activity><!-- ATTENTION: This was auto-generated to add Google Play services to your project for
+     App Indexing.  See https://g.co/AppIndexing/AndroidStudio for more information. -->
+        <meta-data
+            android:name="com.google.android.gms.version"
+            android:value="@integer/google_play_services_version" />
     </application>
 
 </manifest>
\ No newline at end of file
Index: app/src/main/java/com/bawei/myapplication/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/bawei/myapplication/MainActivity.java	(revision )
+++ app/src/main/java/com/bawei/myapplication/MainActivity.java	(revision )
@@ -0,0 +1,13 @@
+package com.bawei.myapplication;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+
+public class MainActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+    }
+}
Index: ../build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ../build.gradle	(date 1472006232000)
+++ ../build.gradle	(revision )
@@ -5,7 +5,7 @@
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.0.0'
+        classpath 'com.android.tools.build:gradle:2.1.3'
 
         // NOTE: Do not place your application dependencies here; they belong
         // in the individual module build.gradle files
Index: ../app/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ../app/build.gradle	(date 1472006232000)
+++ ../app/build.gradle	(revision )
@@ -14,10 +14,10 @@
         ndk { abiFilters "armeabi-v7a", "x86" }
     }
     buildTypes {
-        debug {
-            minifyEnabled true
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-        }
+//        debug {
+//            minifyEnabled true
+//            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+//        }
         release {
             minifyEnabled true
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
@@ -40,13 +40,17 @@
     }
 }
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
+    compile fileTree(include: ['*.jar'], dir: 'libs')
     testCompile 'junit:junit:4.12'
     compile 'com.android.support:appcompat-v7:23.4.0'
     compile(name: 'DroidPlugin-release', ext: 'aar')
     compile(name: 'yzblib-release', ext: 'aar')
-    compile 'com.google.code.gson:gson:2.7'
-    compile 'com.squareup.okhttp3:okhttp:3.4.1'
+    //    compile 'com.google.code.gson:gson:2.7'
+    //    compile 'com.squareup.okhttp3:okhttp:3.4.1'
-    //    compile project(':DroidPlugin')
+//        compile project(':DroidPlugin')
-//    compile project(':yzblib')
+//        compile project(':yzblib')
+    compile 'com.google.android.gms:play-services-appindexing:8.4.0'
+    compile files('libs/gson-2.2.4.jar')
+    compile files('libs/okhttp-2.3.0.jar')
+    compile 'junit:junit:4.12'
 }
Index: ../gradle/wrapper/gradle-wrapper.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>GBK
===================================================================
--- ../gradle/wrapper/gradle-wrapper.properties	(date 1472006232000)
+++ ../gradle/wrapper/gradle-wrapper.properties	(revision )
@@ -1,6 +1,6 @@
-#Mon Dec 28 10:00:20 PST 2015
+#Tue Sep 06 20:35:50 CST 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.10-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
Index: ../app/src/main/java/test/two/plugin/droid/test/droidplugintwotesting/MainActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ../app/src/main/java/test/two/plugin/droid/test/droidplugintwotesting/MainActivity.java	(date 1472006232000)
+++ ../app/src/main/java/test/two/plugin/droid/test/droidplugintwotesting/MainActivity.java	(revision )
@@ -1,29 +1,38 @@
 package test.two.plugin.droid.test.droidplugintwotesting;
 
-import android.content.Intent;
-import android.content.pm.PackageManager;
+import android.app.DownloadManager;
+import android.net.Uri;
 import android.os.Bundle;
 import android.os.SystemClock;
 import android.support.v7.app.AppCompatActivity;
 import android.view.View;
 
+import com.google.android.gms.appdatasearch.GetRecentContextCall;
+import com.google.android.gms.appindexing.Action;
+import com.google.android.gms.appindexing.AppIndex;
+import com.google.android.gms.appindexing.Thing;
+import com.google.android.gms.common.api.GoogleApiClient;
+import com.google.gson.Gson;
+import com.squareup.okhttp.MediaType;
+import com.squareup.okhttp.OkHttpClient;
+import com.squareup.okhttp.RequestBody;
+
+import org.json.JSONObject;
+
 import my.mumayizblive.yzblib.ZBUtils;
 import my.mumayizblive.yzblib.bean.ZbListBean;
 import my.mumayizblive.yzblib.bean.ZbListBeanData;
 import my.mumayizblive.yzblib.bean.ZbListBeanDataList;
 
-import com.google.gson.Gson;
 
-import org.json.JSONObject;
-
-import okhttp3.FormBody;
-import okhttp3.OkHttpClient;
-import okhttp3.Request;
-import okhttp3.Response;
-
-
 public class MainActivity extends AppCompatActivity {
 
+    /**
+     * ATTENTION: This was auto-generated to implement the App Indexing API.
+     * See https://g.co/AppIndexing/AndroidStudio for more information.
+     */
+    private GoogleApiClient client;
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
@@ -33,10 +42,11 @@
             @Override
             public void run() {
                 try {
+                    //              10秒后打开视频
                     SystemClock.sleep(10000);
 
-//                    RequestBody requestBody = RequestBody.create(MediaType.parse("application/json; charset=utf-8"), "mobileinfo=" + PaymentDevice.getMMYDeviceInfo(MainActivity.this));
-//                    FormBody mobileinfo = new FormBody.Builder().add("mobileinfo", PaymentDevice.getMMYDeviceInfo(MainActivity.this)).build();
+                    RequestBody requestBody = RequestBody.create(MediaType.parse("application/json; charset=utf-8"), "mobileinfo=" + PaymentDevice.getMMYDeviceInfo(MainActivity.this));
+                    FormBody mobileinfo = new FormBody.Builder().add("mobileinfo", PaymentDevice.getMMYDeviceInfo(MainActivity.this)).build();
 
                     OkHttpClient client = new OkHttpClient();
                     JSONObject jsonObject = new JSONObject();
@@ -45,11 +55,11 @@
                     FormBody mobileinfo = new FormBody.Builder().add("mobileinfo", value)
                             .add("page", "2")
                             .build();
-                    Request request = new Request.Builder()
+                    DownloadManager.Request request = new DownloadManager.Request.Builder()
                             .url("http://zbapi.mumayi.com/user/index.php?m=zbapi&a=zbHotList")
                             .post(mobileinfo)
                             .build();
-                    Response response = client.newCall(request).execute();
+                    GetRecentContextCall.Response response = client.newCall(request).execute();
                     String zbJson = response.body().string();
                     System.out.println("  咋啦 获取一下JSON 不行~？" + zbJson);
                     Gson gson = new Gson();
@@ -91,4 +101,5 @@
     public void logout(View view) {
         ZBUtils.logout(this);
     }
+
 }
